lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -35,22 +35,22 @@ sub edit_user : Path('/admin/user/edit') Args(1) RequireAuth HiddenOnSlaves
     my $form2 = $c->form(
         form => 'User::EditProfile',
         item => {
-            email            => $user->email,
+            username          => $user->name,
+            email             => $user->email,
             skip_verification => 0,
-            website            => $user->website,
-            biography        => $user->biography
+            website           => $user->website,
+            biography         => $user->biography
         },
     );
 
     if ($c->form_posted) {
-        if ($form->submitted_and_valid($c->req->params )) {
+        if ($form->submitted_and_valid($c->req->params) 
+            && $form2->submitted_and_valid($c->req->params )) {
```

Comment:
Combining the `if`s changes the semantics, unless both forms will always be submitted at the same time. Are you sure that is the case? Why?